src/block/CocoaBlock.php

```diff
@@ -65,8 +64,8 @@ protected function recalculateCollisionBoxes() : array{
 		];
 	}
 
-	public function getSupportType(int $facing) : SupportType{
-		return SupportType::NONE;
+	public function isSolid() : bool{
```

Comment:
I'm not sure what you expect this to do. It's essentially the inverse of Flowable.

Reply:
Sorry, you're right, for some reason I thought solid meant it has collision.